### PR TITLE
Update Lightning import error handling

### PR DESCRIPTION
**Pull request by lilin90**
### What is changed, added or deleted? (Required)

Update the suggested handling method for the data inaccuracy error

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/13793

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ incremental-import = true
 
 2. 如果是不影响数据正确性的报错，如网络超时，可以在每一个失败的节点上使用 tidb-lightning-ctl 工具清除断点续传源数据中记录的错误，然后重启这些异常的节点，从断点位置继续导入，详见 [checkpoint-error-ignore](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-ignore)。
 
-3. 如果是影响数据正确性的报错，如 checksum mismatched，表示源文件中有非法的数据，则需要在每一个失败的节点上使用 tidb-lightning-ctl 工具，清除失败的表中已导入的数据及断点续传相关的源数据，详见 [checkpoint-error-destroy](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-destroy)。此命令会删除下游导入失败表中已导入的数据，因此，应在所有 TiDB Lightning 节点（包括任务正常结束的）重新配置和导入失败的表的数据，可以配置 filters 参数只导入报错失败的表。
+3. 如果是影响数据正确性的报错，如 checksum mismatched，表示源文件中有非法的数据，则需要在每一个 Lightning 节点（无论是否成功导入数据）上使用 tidb-lightning-ctl 工具，清除失败的表中已导入的数据及断点续传相关的源数据，详见 [checkpoint-error-destroy](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-destroy)。此命令会删除下游导入失败的表中已导入的数据、checkpoint、多个并行导入任务的 meta 表信息等。之后在所有 TiDB Lightning 节点（包括任务正常结束的）重新配置和导入失败的表的数据，可以配置 filters 参数只导入报错失败的表。重新配置任务时，注意请不要将 checkpoint-error-destroy 命令放在每一个 Lightning 节点的启动脚本中，否则会多次删除本次并行任务对应的 Meta 表，新建的并行导入任务导入数据时会产生异常行为。
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-distributed-import.md:195]:
```suggestion
3. 如果是影响数据正确性的报错，如 checksum mismatched，表示源文件中有非法的数据，则需要在每一个 Lightning 节点（无论是否成功导入数据）上使用 tidb-lightning-ctl 工具，清除失败的表中已导入的数据及断点续传相关的源数据，详见 [checkpoint-error-destroy](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-destroy)。此命令会删除下游导入失败的表中已导入的数据、checkpoint、多个并行导入任务的 Meta 表信息等。之后你需要在所有 TiDB Lightning 节点（包括任务正常结束的）重新配置和导入失败的表的数据，此时你可以配置 filters 参数只导入报错失败的表。重新配置任务时，注意请不要将 checkpoint-error-destroy 命令放在每一个 Lightning 节点的启动脚本中，否则会多次删除本次并行任务对应的 Meta 表，导致新建的并行导入任务导入数据时会产生异常行为。
```

**Review by ran-huang — APPROVED** [review]

**Comment by lilin90** [comment]
@Frank945946 PTAL

**Review by Frank945946 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 2]
```diff
@@ -190,9 +190,17 @@ incremental-import = true
 
 1. 如果是正常退出(如手动 Kill 等)，或内存溢出被操作系统终止等，可以在适当调整配置后直接重启 TiDB Lightning，无须任何其他操作。
 
-2. 如果是不影响数据正确性的报错，如网络超时，可以在每一个失败的节点上使用 tidb-lightning-ctl 工具清除断点续传源数据中记录的错误，然后重启这些异常的节点，从断点位置继续导入，详见 [checkpoint-error-ignore](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-ignore)。
+2. 如果是不影响数据正确性的报错，例如网络超时，请按以下步骤解决：
 
-3. 如果是影响数据正确性的报错，如 checksum mismatched，表示源文件中有非法的数据，则需要在每一个失败的节点上使用 tidb-lightning-ctl 工具，清除失败的表中已导入的数据及断点续传相关的源数据，详见 [checkpoint-error-destroy](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-destroy)。此命令会删除下游导入失败表中已导入的数据，因此，应在所有 TiDB Lightning 节点（包括任务正常结束的）重新配置和导入失败的表的数据，可以配置 filters 参数只导入报错失败的表。
+    1. 在每一个失败的节点上，使用 tidb-lightning-ctl 工具执行 [checkpoint-error-ignore](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-ignore)命令，清除断点续传源数据中记录的错误。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-distributed-import.md:195]:
```suggestion
    1. 在每一个失败的节点上，执行 [checkpoint-error-ignore](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-ignore) 命令，清除断点续传源数据中记录的错误。
```

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 3]
```diff
@@ -190,9 +190,17 @@ incremental-import = true
...
+
+    2. 重启这些异常的节点，从断点位置继续导入。
+
+3. 如果是影响数据正确性的报错，如 checksum mismatched，表示源文件中有非法的数据，请按以下步骤解决：
+
+    1. 在每一个 Lightning 节点（无论是否成功导入数据）上使用 tidb-lightning-ctl 工具，清除失败的表中已导入的数据及断点续传相关的源数据，详见 [checkpoint-error-destroy](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-destroy)。此命令会删除下游导入失败的表中已导入的数据、相应的 checkpoint、多个并行导入任务的 Meta 表信息等。
+
+    2. 在所有 TiDB Lightning 节点（包括任务正常结束的）重新配置和导入失败的表的数据，此时你可以配置 filters 参数只导入报错失败的表。重新配置任务时，不要将 checkpoint-error-destroy 命令放在每一个 Lightning 节点的启动脚本中，否则会多次删除本次并行任务对应的 Meta 表，导致新建的并行导入任务导入数据时产生异常行为。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-distributed-import.md:203]:
```suggestion
    2. 在所有 TiDB Lightning 节点（包括任务正常结束的）重新配置和导入失败的表的数据，此时你可以配置 [`filter`](/table-filter.md) 参数只导入报错失败的表。重新配置任务时，不要将 checkpoint-error-destroy 命令放在每一个 Lightning 节点的启动脚本中，否则会多次删除本次并行导入任务对应的 Meta 表，导致新建的并行导入任务导入数据时产生异常行为。
```

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 4]
```diff
@@ -188,11 +188,19 @@ incremental-import = true
...
 在并行导入过程中，如果一个或多个 TiDB Lightning 节点异常终止，需要首先根据日志中的报错明确异常退出的原因，然后根据错误类型做不同处理：
 
-1. 如果是正常退出(如手动 Kill 等)，或内存溢出被操作系统终止等，可以在适当调整配置后直接重启 TiDB Lightning，无须任何其他操作。
+- 如果是正常退出(如手动 Kill 等)，或内存溢出被操作系统终止等，可以在适当调整配置后直接重启 TiDB Lightning，无须任何其他操作。
 
-2. 如果是不影响数据正确性的报错，如网络超时，可以在每一个失败的节点上使用 tidb-lightning-ctl 工具清除断点续传源数据中记录的错误，然后重启这些异常的节点，从断点位置继续导入，详见 [checkpoint-error-ignore](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-ignore)。
+- 如果是不影响数据正确性的报错，例如网络超时，请按以下步骤解决：
 
-3. 如果是影响数据正确性的报错，如 checksum mismatched，表示源文件中有非法的数据，则需要在每一个失败的节点上使用 tidb-lightning-ctl 工具，清除失败的表中已导入的数据及断点续传相关的源数据，详见 [checkpoint-error-destroy](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-destroy)。此命令会删除下游导入失败表中已导入的数据，因此，应在所有 TiDB Lightning 节点（包括任务正常结束的）重新配置和导入失败的表的数据，可以配置 filters 参数只导入报错失败的表。
+    1. 在每一个失败的节点上，执行 [checkpoint-error-ignore](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-ignore) 命令，清除断点续传源数据中记录的错误。
```
Comment (ran-huang) [inline tidb-lightning/tidb-lightning-distributed-import.md:195]: checkpoint-error-ignore 的值要设置成什么，是 `all` 还是筛选具体的表？

**Comment by lilin90** [comment]
Made more changes. @Frank945946 @ran-huang PTAL again. Thanks!

File: tidb-lightning/tidb-lightning-distributed-import.md (hunk 4 above)
Comment (lilin90) [reply tidb-lightning/tidb-lightning-distributed-import.md:195]: @Frank945946 这里是用户自己判断吗？

Comment (Frank945946) [reply tidb-lightning/tidb-lightning-distributed-import.md:195]: @lilin90 @ran-huang ran 填 all 就行

Comment (lilin90) [inline tidb-lightning/tidb-lightning-distributed-import.md:195]:
```suggestion
    1. 在每一个失败的节点上，执行 [checkpoint-error-ignore](/tidb-lightning/tidb-lightning-checkpoints.md#--checkpoint-error-ignore) 命令，值设置为 `all`，以清除断点续传源数据中记录的错误。
```

**Review by Frank945946 — APPROVED** [review]

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 5]
```diff
@@ -188,11 +188,19 @@ incremental-import = true
 
 在并行导入过程中，如果一个或多个 TiDB Lightning 节点异常终止，需要首先根据日志中的报错明确异常退出的原因，然后根据错误类型做不同处理：
 
-1. 如果是正常退出(如手动 Kill 等)，或内存溢出被操作系统终止等，可以在适当调整配置后直接重启 TiDB Lightning，无须任何其他操作。
+- 如果是正常退出(如手动 Kill 等)，或内存溢出被操作系统终止等，可以在适当调整配置后直接重启 TiDB Lightning，无须任何其他操作。
```
Comment (lilin90) [inline tidb-lightning/tidb-lightning-distributed-import.md:191]:
```suggestion
- 如果是正常退出，例如手动 Kill 或内存溢出被操作系统终止等，可以在适当调整配置后直接重启 TiDB Lightning，无须任何其他操作。
```

**Pull request merged**